components/Participant.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { createRef, useEffect, useState } from 'react';
+
+const Participant = ({ gain, participant, isYou }) => {
+  const [videoRef] = useState(createRef<HTMLVideoElement>());
+
+  const playTrack = track => {
+    if (track.kind === 'audio') {
+      if (!isYou) {
+        // Don't play your own audio
+        var audio = new Audio();
+        audio.srcObject = postProcessAudioTrack(track);
+      }
+    } else if (track.kind === 'video') {
+      taggedLog('Creating new MediaStream');
+      taggedLog('track:', track);
+      const mediaStreamTrack = track.mediaStreamTrack;
+      const stream = new MediaStream();
+      stream.addTrack(mediaStreamTrack);
+      // @ts-ignore
+      videoRef.current.srcObject = stream;
+    } else {
+      taggedLog('non-audio or video track:', track);
+    }
+  };
+
+  useEffect(() => {
+    participant.tracks.forEach(publication => {
```

Comment:
This is where the data track comes in, too. Right now we ignore it and only do something if `track.kind === 'audio'` or `track.kind === 'video'`.